### PR TITLE
Run codespell to fix spelling

### PR DESCRIPTION
**Pull request by rhatdan**
[NO TESTS NEEDED] Just fixing spelling.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: completions/powershell/podman-remote.ps1 [hunk 1]
```diff
@@ -216,7 +216,7 @@ Register-ArgumentCompleter -CommandName 'podman-remote' -ScriptBlock {
             Default {
                 # Like MenuComplete but we don't want to add a space here because
                 # the user need to press space anyway to get the completion.
-                # Description will not be shown because thats not possible with TabCompleteNext
+                # Description will not be shown because that's not possible with TabCompleteNext
```
Comment (Luap99) [inline completions/powershell/podman-remote.ps1:219]: Please remove this. The completion files are auto generated with cobra.

File: completions/powershell/podman.ps1 [hunk 2]
```diff
@@ -216,7 +216,7 @@ Register-ArgumentCompleter -CommandName 'podman' -ScriptBlock {
             Default {
                 # Like MenuComplete but we don't want to add a space here because
                 # the user need to press space anyway to get the completion.
-                # Description will not be shown because thats not possible with TabCompleteNext
+                # Description will not be shown because that's not possible with TabCompleteNext
```
Comment (Luap99) [inline completions/powershell/podman.ps1:219]: same here

**Comment by baude** [comment]
LGTM

**Comment by edsantiago** [comment]
Oh sweet, the completions check worked:
```
tree is dirty, please run 'make completions' and commit all changes

 M completions/powershell/podman-remote.ps1
 M completions/powershell/podman.ps1
```

File: test/system/255-auto-update.bats [hunk 3]
```diff
@@ -305,7 +305,7 @@ EOF
         fi
     done
 
-    # Only check the last service is started. Previous services should already actived.
+    # Only check the last service is started. Previous services should already activated.
```
Comment (edsantiago) [inline test/system/255-auto-update.bats:308]: Neither of these make sense. Maybe "already **be**" activated"? And, I dislike the word "should": it's ambiguous, sometimes meaning "we expect that", and sometimes meaning "we will take care of it". How about:
```
# Check that only the last service is started; previous services have been activated elsewhere
```
(but even that is cringy. I don't have a good sense for what is being tested here).

File: completions/powershell/podman.ps1 (hunk 2 above)
Comment (rhatdan) [reply completions/powershell/podman.ps1:219]: Remove ps1 from codespell processing.

**Comment by edsantiago** [comment]
@rhatdan I think you accidentally included vendor commits

**Pull request merged**